config.yaml

```diff
@@ -31,9 +31,8 @@ metriccache.check.ids: []
 
 plugin.eventlog.eventlog_url: 'https://eventlog.example.com/'
 
-plugin.history.kairosdb_host: 'localhost'
-plugin.history.kairosdb_port: 8083
-plugin.history.kairosdb_history_enabled: False
+plugin.history.url: 'localhost'
```

Comment:
we should probably default to the Vagrant box settings (port 8083 for KairosDB IIRC)